### PR TITLE
CEXT-1818: Commerce-eventing module events subscriptions

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request (PR) will add additional information about the configuration of subscribed events via the CLI command.
https://jira.corp.adobe.com/browse/CEXT-1818

## Affected pages

- https://developer.adobe.com/commerce/events/get-started/configure-commerce/#subscribe-and-register-events

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/get-started/configure-commerce.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ Commerce provides two sources for events: observers and plugins. You must specif
    ```bash
    bin/magento events:subscribe observer.customer_login --fields=firstname --fields=lastname --fields=entity_id
    ```
+   
+   **Warning**: If you subscribe to events with `events:subscribe` command the configuration will be stored in `app/etc/config.php` and you should keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment so as result the event subscription will be lost as well.
```
Comment (meker12) [inline src/pages/get-started/configure-commerce.md:92]: @oshmyheliuk Does this warning apply to the event provider configuration for a Commerce app deployed on cloud as well?

Comment (oshmyheliuk) [reply src/pages/get-started/configure-commerce.md:92]: No, the event provider configuration is not stored in `app/etc/config.php`, it's stored in the database, so it won't be accidentally erased during redeployment.
This warning is only related to event subscriptions via the CLI command.

Comment (keharper) [inline src/pages/get-started/configure-commerce.md:92]:
```suggestion
   **Warning**: When you use the `events:subscribe` command to subscribe to events on an on-premises installation, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well. Cloud deployments do not store this information in the `config.php`, and event subscriptions are not affected when Commerce is redeployed.
```

File: src/pages/get-started/configure-commerce.md [hunk 2]
```diff
@@ -88,6 +88,9 @@ Commerce provides two sources for events: observers and plugins. You must specif
...
+   To make these changes permanent please add appropriate configuration to the `app/etc/config.php` file under Git.
```
Comment (keharper) [inline src/pages/get-started/configure-commerce.md:93]:
```suggestion
   To make these changes permanent, manually add the appropriate configuration to the `app/etc/config.php` file under Git.
```

**Comment by oshmyheliuk** [comment]
I would change that warning to something like that:
```
When you use the `events:subscribe` command to subscribe to events on a Cloud environment, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well. 
```

File: src/pages/get-started/configure-commerce.md [hunk 3]
```diff
@@ -88,6 +88,9 @@ Commerce provides two sources for events: observers and plugins. You must specif
    ```bash
    bin/magento events:subscribe observer.customer_login --fields=firstname --fields=lastname --fields=entity_id
    ```
+   
+   **Warning**: When you use the `events:subscribe` command to subscribe to events on a Cloud environment, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well.
```
Comment (keharper) [inline src/pages/get-started/configure-commerce.md:92]:
```suggestion
   When you use the `events:subscribe` command to subscribe to events on a Cloud environment, configuration information is stored in the `app/etc/config.php` file. You must keep in mind that this file can be replaced with the `app/etc/config.php` file from Git during deployment. As a result, the event subscription will be replaced as well.
```

**Review by keharper — APPROVED** [review]

**Pull request merged**